docs/src/tool/index.md

```diff
@@ -219,6 +219,18 @@ $ reach scaffold
 
 The files created are the same as those used temporarily by `{!cmd} reach run`.
 
+# {#ref-usage-support} @{ref("cmd", "reach support")} `reach support`
+
+You can upload a gist to GitHub using
+
+```cmd
+$ reach support
+```
+
+Successful completion of this command gives you a link to a gist on GitHub that anyone with the link can view.
+
+`{!cmd} reach support` uploads `index.rsh` from the current directory, if it can find it, and `index.mjs` from the current directory, if it can find it.
```

Comment:
It should say that it takes a list of files that defaults to `index.rsh index.mjs` if none is given

Reply:
The current version doesn't take a list of files.

Reply:
Ah, but you'll update that PR?

Reply:
> [My plan is to add a commit that further updates the documentation to #1063, after successful merging of this pull request, to avoid merge conflicts.](https://github.com/reach-sh/reach-lang/pull/1202#issue-1284730698)

Yes.

[a9caf2a](https://github.com/reach-sh/reach-lang/pull/1063/commits/a9caf2ad8e69f9025424522fa52dde78c1301114)